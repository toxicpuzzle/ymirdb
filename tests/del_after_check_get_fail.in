SET a 1 2 3
SET b 4 5 6
SET c 7 8 9
PUSH a b
PUSH b c
APPEND a c
SNAPSHOT
DEL a 
GET a
FORWARD a
BACKWARD a
POP a 
BACKWARD b
BACKWARD c
bye