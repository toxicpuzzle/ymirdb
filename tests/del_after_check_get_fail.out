> ok

> ok

> ok

> ok

> ok

> ok

> saved as snapshot 1

> ok

> no such key

> no such key

> no such key

> no such key

> nil

> b 

> bye